codeclimate-validate-config/Gemfile.lock

```diff
@@ -0,0 +1,49 @@
+GEM
```

Comment:
This sub-Gemfile was more of a convenience thing: I could `cd codeclimate-validate-config` and suddenly find myself working in a much simpler, but still complete project tree.

The Ruby environment (in the container) is a shared global thing, created by the single top-level `bundle` in the Dockerfile. We could definitely figure things out one way or another, but for now I'd chalk this up to a non-interesting packaging concern.